### PR TITLE
FAT-2742-C356802

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-2742
***
https://foliotest.testrail.io/index.php?/cases/view/356802
***
***

### CODE REVIEW

File: cypress/e2e/data-import/e2e/view-all-filter.cy.js [hunk 1]
```diff
@@ -31,8 +31,8 @@ describe('ui-data-import', () => {
     });
 
     cy.visit(TopMenu.dataImportPath);
-    // TODO delete code after fix https://issues.folio.org/browse/MODDATAIMP-691
-    DataImport.clickDataImportNavButton();
+    // TODO delete reload after fix https://issues.folio.org/browse/MODDATAIMP-691
+    cy.reload();
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/e2e/view-all-filter.cy.js:35]: Fix linter issues

File: cypress/support/fragments/inventory/holdingsRecordView.js [hunk 2]
```diff
@@ -125,6 +125,14 @@ export default {
   checkMarkAsSuppressedFromDiscovery:() => cy.expect(root
     .find(HTML(including('Warning: Holdings is marked suppressed from discovery')))
     .exists()),
+  checkElectronicAccess:(relationshipValue, uriValue) => {
+    cy.expect(Accordion('Electronic access')
+      .find(MultiColumnListCell({ row: 0, columnIndex: 0, content: relationshipValue }))
+      .exists());
+    cy.expect(Accordion('Electronic access')
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/holdingsRecordView.js:132]: move to const

**Review by IhorBohdan — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/data-import/e2e/view-all-filter.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/e2e/view-all-filter.cy.js:35]: Done

File: cypress/support/fragments/inventory/holdingsRecordView.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/inventory/holdingsRecordView.js:132]: Done

**Pull request merged**